src/pages/lib/auth/manage-session/q/platform/[platform].mdx

```diff
@@ -0,0 +1,218 @@
+export const meta = {
+  title: `Manage user session and credentials`,
+  description: `Learn how to manage user sessions and credentials.`,
+};
+
+<InlineFilter filters={["js", "react-native"]}>
+
+Amplify Auth provides access to current user sessions and tokens to help you retrieve your user's information to determine if they are signed in with a valid session and control their access to your app. You can manage tokens and expiration times and revoke sessions. In this guide we will review how to retrieve your user’s session and understand what token management options are available.
+
+Before you begin you will need:
+- An Amplify project with the Auth category configured
+- The Amplify libraries installed and configured
+- A test user signed in
+
+## Retrieve your current authenticated user
+
+You can use the `Auth.currentAuthenticatedUser()` API which returns the full `CognitoUser` object. This object contains information about the currently authenticated user including the session and user attributes.
+
+You can call `Auth.currentAuthenticatedUser()` and get the current authenticated user object:
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+async function currentAuthenticatedUser() {
+  try {
+    const user = await Auth.currentAuthenticatedUser({
+      bypassCache: false // Optional and is false by default. If set to true, this call 
+      // will send a request to Cognito to get the latest user data.
+    });
+    console.log(user);
+  } catch(err) {
+    console.log(err);
+  }
+};
+```
+</Block>
+<Block name="JavaScript">
+
+```javascript
+import { Auth } from 'aws-amplify';
+
+async function currentAuthenticatedUser() {
+  try {
+    const user = await Auth.currentAuthenticatedUser({
+      bypassCache: false // Optional and is false by default. If set to true, this call
+      // will send a request to Cognito to get the latest user data.
+    });
+    console.log(user);
+  } catch(err) {
+    console.log(err);
+  }
+};
+```
+</Block>
+</BlockSwitcher>
+
+This method can be used to check if a user is signed in as it will throw an error if there is no current user signed in. 
+
+## Retrieve your user’s session
+
```

Comment:
@abdallahshaban557 I worked on simplifying this intro a bit but think it can use another pass. Once we are happy with this opening this can go to tech review as all other changes from your first review are done.

Reply:
This method can be used to check if a user is signed. It returns an error if user is not signed in.